migrations/versions/0321_drop_postage_constraints.py

```diff
@@ -0,0 +1,28 @@
+"""
+
+Revision ID: 0321_drop_postage_constraints
+Revises: 0320_optimise_notifications
+Create Date: 2020-06-08 11:48:53.315768
+
+"""
+import os
+
+from alembic import op
+
+
+revision = '0321_drop_postage_constraints'
+down_revision = '0320_optimise_notifications'
+environment = os.environ['NOTIFY_ENVIRONMENT']
+
+
+def upgrade():
+    if environment not in ["live", "production"]:
+        op.execute('ALTER TABLE notifications DROP CONSTRAINT IF EXISTS chk_notifications_postage_null')
+        op.execute('ALTER TABLE notification_history DROP CONSTRAINT IF EXISTS chk_notification_history_postage_null')
+
+    op.execute('ALTER TABLE templates DROP CONSTRAINT IF EXISTS chk_templates_postage')
+    op.execute('ALTER TABLE templates_history DROP CONSTRAINT IF EXISTS chk_templates_history_postage')
+
+
+def downgrade():
+    pass
```

Comment:
how come we're not re-adding them here?

Reply:
Just because you can't add them back in a single migration that is safe to run in production. I suppose it is possible to do a `if environment not in ["live", "production"]:` in the downgrade though - what do you think?

Reply:
Now updated to add in a downgrade that does not run in production